services/couchdb/Dockerfile

```diff
@@ -1 +0,0 @@
-FROM couchdb:latest
```

Comment:
Gradle doesn't even build this. Ansible just deploys a plain image from Dockerhub.